config/7.3.0/fapi1part2adv/ig/routes/routes-service/20-as-dcr-endpoint.json

```diff
@@ -13,39 +13,17 @@
           "type": "FapiDcrFilterChain",
           "config": {
             "auditService" : "fapiAuditService",
+            "forwardedHost" : "&{identity.platform.fqdn}",
             "clientCertificate": "${pemCertificate(urlDecode(request.headers['ssl-client-cert'][0]))}",
             "trustedDirectoryService": "TrustedDirectoryService",
             "jwkSetService": "JwkSetService",
             "supportedSigningAlgorithms": ["PS256"],
-            "supportedTokenEndpointAuthMethods": "${oauth2.tokenEndpointAuthMethodsSupported}"
-          }
-        },
-        {
-          "name": "ManageApiClientFilter",
-          "type": "ManageApiClientFilter",
-          "config": {
+            "supportedTokenEndpointAuthMethods": "${oauth2.tokenEndpointAuthMethodsSupported}",
             "apiClientService": "IdmApiClientService",
             "apiClientOrgService": "IdmApiClientOrganisationService",
             "clientId": "${request.queryParams.client_id[0]}"
           }
         },
-        {
-          "comment": "Set Host header for downstream",
-          "name": "HeaderFilter-ChangeHostToIAM",
```

Comment:
How does that relate to the new filter in IG ?

Reply:
I cleaned up stuff already migrated at the same time.